nsxt/policy_common.go

```diff
@@ -201,7 +201,6 @@ func getSecurityPolicyAndGatewayRulesSchema(scopeRequired bool, isIds bool, nsxI
 			Description:  "IP version",
 			Optional:     true,
 			ValidateFunc: validation.StringInSlice(securityPolicyIPProtocolValues, false),
-			Default:      model.Rule_IP_PROTOCOL_IPV4_IPV6,
```

Comment:
I believe this should be set to Computed, since it would be initialized by platform now.